13/umbraco-ui-builder/collections/retrieve-child-collections.md

```diff
@@ -0,0 +1,70 @@
+---
+description: Configuring **one-to-many** relationships in Umbraco UI Builder, the backoffice UI builder for Umbraco.
```

Comment:
This is copied from the other page, so needs adjusting to this one.

Reply:
Updated